api.py

```diff
@@ -170,6 +174,29 @@ def assemble_release_data(version_data: str):
             except:
                 release_github_version = -1
 
+        # version code
+        # get git hash from release page
+        tags = elem.cssselect(".release > .release-meta > .tag-references a code")
+        if len(tags) == 0:
+            release_github_version_code = -1
+        else:
+            try:
+                release_github_hash = tags[0].text
+
+                # use git hash to get the matching build.gradle file
+                response = yield tornado.gen.multi((
```

Comment:
I just invested five minutes to get this done. Therefore code quality is lacking behind 😅 
I do not have time to improve this in the next days, but wanted to get this out before Friday.
This is rubbish, but I couldn't get it done better and being that fast. Can you please take a look @TheAssassin? If you do not have time, I'll imrpove this on Friday.
``` phython
gradle_file_data = [x.body for x in response]
gradle_file_data = gradle_file_data[0]
```